app/src/main/java/com/concordium/wallet/ui/walletconnect/WalletConnectService.kt

```diff
@@ -134,10 +135,19 @@ class WalletConnectService : Service(), SignClient.WalletDelegate, CoreClient.Co
     }
 
     private fun pairWC(wcUri: String) {
-
-        println("LC -> CALL PAIR $wcUri")
+        if (matchesURLScheme(wcUri).not()) {
```

Comment:
Is it even possible to end up here without the URL matching the scheme? Wouldn't it just not trigger the app?